src/components/Controls/ToogleScreenShareButton/ToggleScreenShareButton.tsx

```diff
@@ -28,28 +33,37 @@ export default function ToggleScreenShareButton(props: { disabled?: boolean }) {
   const screenShareParticipant = useScreenShareParticipant();
   const { room } = useVideoContext();
   const disableScreenShareButton = screenShareParticipant && screenShareParticipant !== room.localParticipant;
-  const tooltipMessage = isScreenShared ? 'Stop Sharing Screen' : 'Share Screen';
+  const screenShareIsSupported = navigator.mediaDevices && navigator.mediaDevices.getDisplayMedia;
```

Comment:
Nit: We usually do verb-first convention for booleans. How about `isScreenShareSupported`?

Reply:
Fixed - thanks!